src/lerobot/transport/utils.py

```diff
@@ -66,7 +66,7 @@ def send_bytes_in_chunks(buffer: bytes, message_class: Any, log_prefix: str = ""
     logging_method(f"{log_prefix} Published {sent_bytes / 1024 / 1024} MB")
 
 
-def receive_bytes_in_chunks(iterator, queue: Queue, shutdown_event: Event, log_prefix: str = ""):  # type: ignore
+def receive_bytes_in_chunks(iterator, queue: Queue | None, shutdown_event: Event, log_prefix: str = ""):  # ruff: noqa
```

Comment:
```suggestion
def receive_bytes_in_chunks(iterator, queue: "Queue | None", shutdown_event: Event, log_prefix: str = ""):  # ruff: noqa
```

We actually have to add quotes here. Otherwise the exception is still thrown. https://github.com/huggingface/lerobot/pull/1521/commits/5c128aed5cdbbd78f696c0936e961fd07bc636c1

Reply:
Ok, I see. The pyupgrade change it in any case. I gave up fightimg with it - changed type annotation from `mp.Queue` to just `Queue`, now it should be fine.